settings/src/lib.rs

```diff
@@ -99,41 +101,84 @@ impl Default for PaymentSettings {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+pub struct ExitServer {
+    pub id: Identity,
+    pub registration_port: u16,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub our_details: Option<ExitClientDetails>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub general_details: Option<ExitDetails>,
```

Comment:
How's calling this "general" details descriptive of the field's role? Why are ExitClientDetails stored here and why is it called `our_details` when it's obviously not the exit server's details in that field?

Reply:
`ExitServer` is a struct which represents an exit server on the client, `our details` mean details for the exit which is specific to the client and `general_details` means details which is the same for all clients of that particular exit. Any suggestions for less confusing names?

Reply:
You're right, `our_details` makes sense. How about just `details` for `general_details`?